src/queryCache.js

```diff
@@ -419,6 +421,11 @@ export function makeQueryCache() {
 
 export function defaultQueryReducer(state, action) {
   switch (action.type) {
+    case actionResetFailureCount:
+      return {
+        ...state,
+        failureCount: 0
+      }
```

Comment:
Instead of using this new action, just updated the success action below.

Reply:
The `actionSuccess` handler can look like this: 

```diff
case actionSuccess:
      return {
        ...state,
        status: statusSuccess,
        data: action.data,
        error: null,
        isStale: false,
        isFetching: false,
        canFetchMore: action.canFetchMore,
        updatedAt: Date.now(),
+        failureCount: 0
      }
```

Reply:
that s is a neat, thanks for reviewing my code :pray: